TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/Index.cshtml

```diff
@@ -3,3 +3,95 @@
 @{
     ViewBag.Title = "All records";
 }
+
+<h1 class="govuk-heading-l">@ViewBag.Title</h1>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <div class="govuk-!-margin-bottom-6">
+            <form trs-action="l => l.Persons()" data-testid="search-form">
+                <div class="moj-search trs-search">
+                    <govuk-input asp-for="Search"
+                                 input-class="moj-search__input"
+                                 label-class="moj-search__label govuk-!-font-weight-bold"
+                                 id="search"
+                                 name="search"
+                                 data-testid="search-input"
+                                 type="search">
+                        <govuk-input-hint class="moj-search__hint">Search by: Name, TRN or date of birth (DD/MM/YYYY)</govuk-input-hint>
+                    </govuk-input>
+                    <govuk-button class="moj-search__button" type="submit">Search</govuk-button>
+                </div>                
+            </form>
+        </div>
+    </div>
+</div>
+@if (!string.IsNullOrEmpty(Model.Search))
+{
+    <div class="govuk-grid-row" data-testid="search-results">
+        <div class="govuk-grid-column-full">
+            @if (Model.TotalKnownPages > 1)
+            {
+                <govuk-pagination>
+                    @if (Model.PreviousPage.HasValue)
+                    {
+                        <govuk-pagination-previous asp-page="Index" asp-route-search="@Model.Search" asp-route-pagenumber="@Model.PreviousPage" />
```

Comment:
Can we use `href` here with `LinkGenerator` instead of the `asp-` attributes?

Reply:
TBH I did that in first place and it didn't seem to want to pass the query parameters to the model

Reply:
Can't do at the moment due to bug